engine/script/src/script_sys.cpp

```diff
@@ -495,6 +495,7 @@ union SaveLoadBuffer
      *
      * Returns a table with system information.
      * @name sys.get_sys_info
+     * @param ignore_secure_values [type:boolean] this flag ignores values might be secured by OS e.g. `device_ident`
```

Comment:
I'm no sure if I'm overly cautious here but maybe we should use an option table instead, in case we wish to add some more option in the future?

```
sys.get_sys_info({ ignore_secure = true })
```

Reply:
No I use table with options